sprite.go

```diff
@@ -69,9 +69,7 @@ type Sprite interface {
 	Bounds() *mathf.Rect2
 	ChangeEffect(kind EffectKind, delta float64)
 	ChangeHeading(dir float64)
-	ChangePenColor(delta float64)
-	ChangePenHue(delta float64)
-	ChangePenShade(delta float64)
+	ChangePen(kind PenColorKind, delta float64)
```

Comment:
PR contains changes from #554. We gonna re-review this after #554 being updated.